data/announcements/scheduled.yaml

```
@@ -213,6 +213,7 @@ announcements:
   endTime: 2024-02-01T00:00:00
   style: "background: #d95e00"
   title: Changes to the location of Linux packages for Kubernetes
+  dismissable: True
```

```suggestion
  dismissable: true
```

I would hold this PR, get reviews, and then omit the change to the banner from _this_ PR. Then, once the feature is merged, mark some banners as dismissable.

Maybe even do a selection of the old ones, in case people come along and start a copy-and-paste.